src/elements/content-sidebar/activity-feed/task-new/TaskStatus.js

```diff
@@ -29,33 +36,31 @@ const StatusIcon = ({ status, ...rest }: { status: TaskStatus }) => {
             return <IconReject width={ICON_SIZE} height={ICON_SIZE} {...rest} />;
         case TASK_NEW_NOT_STARTED:
         case TASK_NEW_IN_PROGRESS:
-            return <IconPending width={ICON_SIZE} height={ICON_SIZE} {...rest} />;
```

Comment:
@alexkrolick , just to confirm: Pending status is no longer needed? It was added recently, iirc.

Reply:
correct, it doesn't show an icon for the initial states